### DISCUSSION TITLE
AppIntro 7.0.0 Beta

### DISCUSSION THREAD

**Discussion by paolorotolo** [AppIntro/AppIntro#1119] [General]
This is a new major release of AppIntro. Please note that this release contains multiple new features (see below), several bugfixes, as well as multiple breaking changes.

### Summary of Changes
* [#1112] We migrated AppIntro from ViewPager to ViewPager2

### Breaking Changes
* We removed `setScrollDurationFactor` since customizing scroll duration is supported anymore on ViewPager2
* `setCustomTransformer` now accepts a `ViewPager2.PageTransformer` instead of `ViewPager.PageTransformer`

**Full Changelog**: https://github.com/AppIntro/AppIntro/compare/6.3.0...7.0.0-beta02

<hr /><em>This discussion was created from the release <a href='https://github.com/AppIntro/AppIntro/releases/tag/7.0.0-beta02'>AppIntro 7.0.0 Beta 2</a>.</em>

**Comment by mveroukis** [comment]
Will this ever be completed as a proper release?
